pal/inc/tlsio_arduino.h

```diff
@@ -32,7 +32,7 @@ MOCKABLE_FUNCTION(, const IO_INTERFACE_DESCRIPTION*, tlsio_arduino_get_interface
     TLSIO_ARDUINO_STATE_CLOSING,    \
     TLSIO_ARDUINO_STATE_ERROR,      \
     TLSIO_ARDUINO_STATE_NULL
-DEFINE_ENUM(TLSIO_ARDUINO_STATE, TLSIO_ARDUINO_STATE_VALUES);
+MU_DEFINE_ENUM(TLSIO_ARDUINO_STATE, TLSIO_ARDUINO_STATE_VALUES);
```

Comment:
It would be good to use the other macro, MU_DEFINE_ENUM_WITHOUT_INVALID so you have less trouble.

Reply:
Here and anywhere else we are defining enums

Reply:
Done.